a.css

```diff
@@ -0,0 +1,146 @@
+header {
+  background-image: url(image/Group.png);
+  background-repeat: no-repeat;
+  background-position: right 80mm;
+  background-color: #1c1a19;
+  height: 350px;
+  width: 375px;
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  position: absolute;
+  margin-left: 933px;
```

Comment:
Again as requested by the previous reviewer your page is not responsive. Kindly remove that left margin inline `16` as it is pushing your whole page out of view in mobile view.